common/ChooseGame.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
You have some merge issues: I renamed this file to ChooseGameScreen.tsx a while back.
Please clean this up :)

Reply:
weird as i pulled the upstream main before i created the pr. as you can see i merged branch into main commit.

Reply:
pulled recent changes again and awaiting rereview.